gamemodes/terrortown/gamemode/shared/lang/english.lua

```diff
@@ -1308,5 +1308,9 @@ L.decoy_help_pri = "Plant the Decoy."
 
 L.set_cross_lines_enable = "Enable crosshair lines"
 
+L.pickup_error_spec = "You cannot pickup this as a spectator."
+L.pickup_error_owns = "You cannot pickup this because you already own the same weapon."
+L.pickup_error_noslot = "You cannot pickup this because you have no free slot available."
```

Comment:
```suggestion
L.pickup_error_spec = "You cannot pick this up as a spectator."
L.pickup_error_owns = "You cannot pick this up because you already have this weapon."
L.pickup_error_noslot = "You cannot pick this up because you have no free slot available."
```
or
```suggestion
L.pickup_error_spec = "You cannot pick up this as a spectator."
L.pickup_error_owns = "You cannot pick up this because you already have this weapon."
L.pickup_error_noslot = "You cannot pick up this because you have no free slot available."
```
Notes:
1) "Own" and "Have" have different meanings.
2) "Pickup" and "Pick up" are not the same.

Reply:
Thank you :+1:

Reply:
> Thank you
Don't forget about other languages!